plugins/reporters/opossum/src/main/kotlin/OpossumReporter.kt

```diff
@@ -280,15 +350,15 @@ class OpossumReporter(
         }
 
         private fun addPackageRoot(id: Identifier, path: String, level: Int = 0, vcs: VcsInfo = VcsInfo.EMPTY) {
-            val mapOfId = packageToRoot.getOrPut(id) { sortedMapOf() }
+            val mapOfId = packageToRoot.getOrPut(id) { mutableMapOf() }
```

Comment:
Forgot to explain. It is actually not unrelated. Kotlinx does not seem to support serialization of the Java type that is used to implement the interface (`TreeMap` and `TreeSet`). Since it is not necessary to sort, I removed it.

Reply:
I see. We're anyway as part of [this](https://github.com/oss-review-toolkit/ort/issues/6235) aiming to not sort in memory, but only on serialization. For collections I've added [this](https://github.com/Kantis/ks3/pull/117/files) to Ks3 a while ago, but sorted maps are indeed still missing, I believe.

Reply:
Ah, nice. I guess since we do not really rely on this, it is simpler to remove it entirely.

Reply:
> it is simpler to remove it entirely.

Simpler for sure, and also ok if it's not likely that humans are looking manually at the report, which is the case in which sorting would be beneficial.

Reply:
I agreed. I think this files are meant to consumed through OpossumUI. And since it is very cumbersome to read them without OpossumUI hopefully everybody will do so.